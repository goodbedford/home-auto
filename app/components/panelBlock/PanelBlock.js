import React from "react";

const PropTypes = React.PropTypes;
const propTypes = {
  msg: PropTypes.string
};
const PanelBlock = (props) => {
  return (
    <div className="home-auto__panel-block">
      <span className="txt txt-lg">{props.msg}</span>
      {props.children}
    </div>
  );
};
PanelBlock.propTypes = propTypes;

export default PanelBlock;
