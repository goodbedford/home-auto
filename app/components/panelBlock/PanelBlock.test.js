/* eslint-disable no-undef */

import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import PanelBlock from "./PanelBlock";

describe("<PanelBlock />", () => {
  const props = {msg: "hello"};

  it("PanelBlock snapshot", () => {
    const wrapper = shallow(<PanelBlock {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});
