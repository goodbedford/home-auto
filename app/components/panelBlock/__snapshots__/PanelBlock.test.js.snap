// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PanelBlock /> PanelBlock snapshot 1`] = `
<div
  className="home-auto__panel-block"
>
  <span
    className="txt txt-lg"
  >
    hello
  </span>
</div>
`;
